### DISCUSSION TITLE
浏览器

### DISCUSSION THREAD

**Discussion by Sunbreak** [deapplization/awesome-switching#4] [General]
### Chromium

下载地址：https://download-chromium.appspot.com/
开源地址：https://www.chromium.org/
优点：开源，兼容性好
缺点：项目被Google主导，对其他浏览器形成事实垄断

### Firefox

下载地址：https://www.mozilla.org/en-US/firefox/
开源地址：-
优点：最注重隐私
缺点：相对小众

**Comment by huo-ju** [comment]
https://github.com/deapplization/awesome-switching/blob/main/README.md#Browser 已经增加到浏览器分类
